File: code-recipes/java/src/test/java/easyship/SubmitFeedRequestRecipeTest.java
package easyship;

import util.RecipeTest;

import java.util.List;

public class SubmitFeedRequestRecipeTest extends RecipeTest {

    protected SubmitFeedRequestRecipeTest() {
        super(
                new SubmitFeedRequestRecipe(),
                List.of(
                    "feeds-createFeedDocument",
                    "feeds-createFeed",
                    "feeds-createFeed"
                )
        );
    }
}

File: code-recipes/php/test/easyship/SubmitFeedRequestRecipeTest.php
<?php

namespace Test\easyship;

use Src\easyship\SubmitFeedRequestRecipe;
use Test\util\RecipeTestCase;

class SubmitFeedRequestRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new SubmitFeedRequestRecipe();
        $this->responses = [
            "feeds-createFeedDocument",
            "feeds-createFeed",
            "feeds-createFeed"
        ];
    }
}
